File: AkitaModel/Product.cs
namespace Skyware.Lis.AkitaModel;

/// <summary>
/// Price list entry
/// </summary>
public class Product
{

    public int Id { get; set; }

    public int DepartmentId { get; set; }

    public int? TestId { get; set; }

    public int? ProfileId { get; set; }

    public string Name { get; set; }

    public decimal Price { get; set; }

    public string PriceText { get; set; }

}

File: AkitaMotelTS/src/Product.ts
/**
 * Skyware.Lis.AkitaModel
 */

/**
 * Price list entry.
 */
export class Product {
  id!: number;
  departmentId!: number;
  rank?: number;
  testId?: number;
  profileId?: number;
  loincCode?: string;
  name?: string;
  price!: number;
  priceText?: string;
}
